hitch-2011/hitch-fe
# Fix/travis branches

## Is this a fix or a feature?

This is a fix.

## What is the change?

Add a 
`on:
    branch: main` to the `.travis.yml` file.
Also change version of package.lock file

Also added node version '12' to `travis.yml` file

## What does it fix?

Fixes issue of Travis CI not passing due to differences in Node versions, and allows for deployment after merging branches to main

## Where should the reviewer start?

In the `travis.yml` file

## How should this be tested?

All tests should pass in Cypress after running `npm cypress: run` and Travis CI should have a passing build for this branch.

## Relevant ScreenShots
